middleware.ts

```diff
@@ -44,6 +46,43 @@ const loadSession = async (request: NextRequest, sessionToken?: string) => {
   }
 };
 
+async function getDefaultWorkspaceId(accessToken: string) {
+  const searchParams = new URLSearchParams({ limit: "1" });
```

Comment:
In the future once we have personal workspaces, a user's personal workspace's ID will be the default workspace ID.